src/main/java/cz/cvut/kbss/termit/config/WebAppConfig.java

```diff
@@ -153,27 +153,27 @@ public SimpleUrlHandlerMapping sparqlQueryControllerMapping() throws Exception {
     }
 
     @Bean
-    public HttpMessageConverter<?> stringMessageConverter() {
+    public HttpMessageConverter<?> termitStringHttpMessageConverter() {
```

Comment:
Here, it was named `stringMessageConverter`, but it is `HttpMessageConverter`; since I was adding `MessageConverter`, I needed to make it different, and naming it `httpStringMessageConverter` conflicts with beans from Spring autoconfiguration.

Reply:
Ok, leave it like this then.